docs/specification/2025-03-26/basic/authorization.mdx

```diff
@@ -8,236 +8,344 @@ title: Authorization
 
 ### 1.1 Purpose and Scope
 
-The Model Context Protocol provides authorization capabilities at the transport level,
-enabling MCP clients to make requests to restricted MCP servers on behalf of resource
-owners. This specification defines the authorization flow for HTTP-based transports.
-
-### 1.2 Protocol Requirements
+The Model Context Protocol provides authorization capabilities at the transport level, enabling MCP
+clients to make requests to restricted MCP servers on behalf of resource owners. This specification
+defines the authorization flow for HTTP-based transports.
+
+The scope of this document is **user authorization** - how a client can authenticate a user to access a
+server and the server determine whether to authorize the user to access its tools, resources, or any
+other capabilities.
+
+This document **does not discuss server-to-server authorization patterns**.
+
+### 1.2 Baseline tenets
+
+1. **MCP server developers are not security experts**. The current specification assumes that developers
+   will be building their own authorization server. That is not something they ever need to do in the
+   context of MCP servers. This also introduces room for serious mistakes that can lead to compromise.
+1. **MCP clients are Authorization Server (AS)-agnostic**. We do not and will not expect clients to be
+   "enlightened" in any capacity to support a protected MCP server. This will fragment the ecosystem.
+   The specification explicitly assumes that clients are not enabled in any unique way to support
+   specific Authorization Servers.
+    - **Note:** It is acceptable for clients to be enhanced for specific Authorization Servers,
+      primarily through pre-registration.
+1. **User Authentication is not a MCP Server concern.** The MCP server can guide the client
+   to the AS but will not be responsible for authenticating the user and obtaining credential
+   artifacts for the user. The MCP server might transform incoming credential artifacts, like tokens, but
```

Comment:
This mention of transforming credential artifacts is confusing. I would just remove it. There isn't any transformation of tokens involved in this really.